python/pysmurf/client/util/smurf_util.py

```diff
@@ -781,7 +796,9 @@ def stream_data_off(self, write_log=True):
 
         
     def read_stream_data(self, datafile, channel=None,
-                         n_samp=None, array_size=None):
+                         n_samp=None, array_size=None,
+                         return_header=False,
+                         write_log=True):
```

Comment:
Docstring needs entry for return_header and write_log.

Reply:
This is resolved in a later commit 

https://github.com/slaclab/pysmurf/blob/issue276/python/pysmurf/client/util/smurf_util.py#L866